### DISCUSSION TITLE
Discord.py is no longer maintained, what now?

### DISCUSSION THREAD

**Discussion by DenverCoder1** [EddieHubCommunity/support#2300] [General]
The maintainer of [discord.py](https://github.com/Rapptz/discord.py) quit and the library will no longer be maintained :scream: 

This is bad news for the Python Discord community, but fortunately members of the dpy community have made forks to continue discord.py and maintain it.

Many forks are rushing to add features, making the library unmaintainable and breaking compatibility.

On the other hand, [Nextcord](https://github.com/nextcord/nextcord) is taking the time to organize the library better to add interaction commands without causing breaking changes and making it compatible with old style commands as well.

[Here](https://www.youtube.com/watch?v=lzyd5wHzDmU) I have made a video describing how to migrate a bot from discord.py to Nextcord.

I hope to continue this series to help everyone learn to make awesome bots in Python!

(I am not affiliated with Nextcord in any way, I simply trust this project more than the others based on the changes and discussions I've seen.)

The other two forks gaining significant traction are https://github.com/Pycord-Development/pycord and https://github.com/iDevision/enhanced-discord.py. Feel free to check out either of these in addition to Nextcord and choose whatever you like.

**Comment by KukretiShubham** [comment]
Glad you shared this info. Thanks
